build.gradle

```diff
@@ -449,14 +449,14 @@ dependencies {
     exclude group: "org.hamcrest", module: "hamcrest-library"
   }
 
-    testCompile group: 'io.rest-assured', name: 'rest-assured', version: '4.0.0'
-    testCompile group: 'io.rest-assured', name: 'rest-assured-common', version: '4.2.0'
+    testCompile group: 'io.rest-assured', name: 'rest-assured', version: '4.3.3'
+    testCompile group: 'io.rest-assured', name: 'rest-assured-common', version: '4.3.3'
```

Comment:
same version in constants hence will update both io.rest-assured in future.

Reply:
Done